sdk/identity/identity/src/az_identity_client_secret_credential.c

```diff
@@ -85,10 +87,16 @@ _az_identity_client_secret_credential_ms_oauth2_send_get_token_request(
   AZ_RETURN_IF_FAILED(az_http_request_builder_init(
       &hrb, hrb_buf, (uint16_t)auth_url.size, AZ_HTTP_METHOD_VERB_POST, auth_url, auth_body));
 
+  static az_http_policy_retry_options default_retry_policy = {
```

Comment:
auth_retry_policy_options is a better name for this structure.  Its not default because its specific to the auth pipeline